azurelinuxagent/common/osutil/default.py

```diff
@@ -299,12 +301,19 @@ def is_current_instance_id(self, id_that):
     @staticmethod
     def is_cgroups_supported():
         """
-        Enabled by default; disabled in WSL/Travis
+        Enabled by default; disabled in WSL/Travis (Disabled on Trusty only)
         """
         is_wsl = '-Microsoft-' in platform.platform()
         is_travis = 'TRAVIS' in os.environ and os.environ['TRAVIS'] == 'true'
+
+        supported_in_travis = True  # By default supported in Travis
+
+        # Travis with python 2.6 fails (on Trusty based systems)
+        if is_travis and PY_VERSION_MAJOR == 2 and PY_VERSION_MINOR == 6:
```

Comment:
This would be removed once validated.

Reply:
The check should not be based on the Python version, but on the platform (trusty). Also, if all the other platforms Travis runs on support cgroups, then we should remove the check on Travis altogether.